js/quote-generator.js

```diff
@@ -0,0 +1,55 @@
+const randomBtn = document.querySelector("#random-btn");
+const allAuthorsBtn = document.querySelector("#all-authors-btn");
+const quote = document.querySelector(".quote");
+let quoteAuthor = document.querySelector(".quote-author");
+let quoteGenre = document.querySelector(".quote-genre");
+const allAuthorsContainer = document.querySelector(".all-authors-container");
+const gridContainer = document.querySelector(".grid-container");
+const topButtons = document.querySelector(".top-buttons");
+const footer = document.querySelector("footer");
+
+async function getRandomQuote() {
+  const promise = await fetch("https://api.quotable.io/random");
```

Comment:
What if we have an error here ? Imagine server is down 

we need to handle this case